src/native/libs/System.Globalization.Native/pal_icushim.c

```diff
@@ -481,6 +481,10 @@ int32_t GlobalizationNative_LoadICU()
     FOR_ALL_ICU_FUNCTIONS
     ValidateICUDataCanLoad();
 
+#if defined(USE_ICUSHIM_STATIC_BUILD)
```

Comment:
Is this needed?

Reply:
Yes, the issue is that the entrypoint is different, dotnet uses `GlobalizationNative_LoadICU` instead of `GlobalizationNative_LoadICUData`

Reply:
I am not able to find the place where `USE_ICUSHIM_STATIC_BUILD` is defined. Where is it defined?

Reply:
I changed it to STATIC_SHIM_COMPILE in latest revision, it is in CMakeLists-standalone.txt

Reply:
You are right, I need to remove it from line 484 as well, thanks to @Beau-Gosse-dev for pointing out.